docs/docs/for-developers/building.md

```diff
@@ -6,19 +6,29 @@ sidebar_label: Building from Source
 
 # Building the Editor
 
+Oni2 can be built from source using the following setup and steps!
+
+This takes around 30-40 mins on an "average" machine, as we need to setup a brand new
+set of tooling for both OCaml and Skia, which are both fairly large. Repeat builds
+should take only a few seconds, as build artifacts are cached by `esy`.
```

Comment:
Thanks for adding this - it's helpful to set expectations on the first-time build experience is.